coding/src/templatize.py

```diff
@@ -27,6 +27,7 @@
         'join_category_description': lambda cat, desc: f'{cat} ({desc})', # function to join category and description in prompt
         'use_description': False, # whether to use description in prompt
         'category_to_description': nyt_descriptions, # dictionary mapping category to descriptio
+        'exemplar_method':'random', # method to select exemplars
```

Comment:
Do you think this is best as in the args dictionary? Or as passed down argument?

Reply:
I want to have this in the args dictionary, because of how I did `set_exemplar_method` and how I did `load_exemplar_candidates`, we can talk about it.